src/App.css

```diff
@@ -1,16 +1,17 @@
 .App-header {
-  background-color: #222;
-  padding-bottom: 0.5rem;
-  color: white;
-  width: 100%;
+    background-color: #222;
```

Comment:
Noticed that some indentations were 2 and some were 4 spaces so I made them all 4 spaces

Reply:
Indentation is probably another area where we're going to have consitency problems. It looks like vite generates code with 2 spaces. I usually only use 4 spaces with python code, myself. We have some code that uses 2 spaces in the repo code, but 4 spaces when displayed in Learn. 😩

Reply:
I'd probably keep it at two. The most important thing is that at least within a file it should be consistent, so the fixes you did to TimelineEvent.css were sorely needed. But otherwise, I'd tend to leave indentation alone.